deduplication/src/main/java/gov/cdc/nbs/deduplication/duplicates/mapper/PersonMergeDataMapper.java

```diff
@@ -0,0 +1,261 @@
+package gov.cdc.nbs.deduplication.duplicates.mapper;
+
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.*;
+
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.lang.NonNull;
+import org.springframework.lang.Nullable;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import gov.cdc.nbs.deduplication.duplicates.model.PersonMergeData;
+import gov.cdc.nbs.deduplication.duplicates.model.PersonMergeData.Address;
+import gov.cdc.nbs.deduplication.duplicates.model.PersonMergeData.Identifier;
+import gov.cdc.nbs.deduplication.duplicates.model.PersonMergeData.Name;
+import gov.cdc.nbs.deduplication.duplicates.model.PersonMergeData.Race;
+import gov.cdc.nbs.deduplication.duplicates.model.PersonMergeData.Telecom;
+
+
+
+public class PersonMergeDataMapper implements RowMapper<PersonMergeData> {
+
+  private static final Map<String, String> RACE_MAP = Map.ofEntries(
+      Map.entry("1002-5", "AMERICAN_INDIAN"),
+      Map.entry("2028-9", "ASIAN"),
+      Map.entry("2054-5", "BLACK"),
+      Map.entry("2076-8", "HAWAIIAN"),
+      Map.entry("2106-3", "WHITE"),
+      Map.entry("2131-1", "OTHER"),
+      Map.entry("U", "UNKNOWN")
+  );
```

Comment:
Possible issue: These values come from the `code_value_general` table. They are `SYS` controlled values so they shouldn't change frequently (user can't edit them through UI), but it is still possible for them to change. 

I'm fine with caching these values but it would be better to pull them from the database on startup.

If we want to keep it this way for now, I'm ok with it. We would just need to put a story in the backlog to address it in the future.